libraries/lib-stretching-sequence/tests/StretchingSequenceIntegrationTest.cpp

```diff
@@ -0,0 +1,79 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
This test file fails to compile in the commit that introduces it.  The Audacity executable compiles.

Reply:
Makes sense, there must be modifications in `WaveTrack` needed for this integration test that I added in follow-up commits.